packages/openneuro-components/src/facets/FacetRadio.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import { AccordionTab } from '../accordion/AccordionTab'
+import { AccordionWrap } from '../accordion/AccordionWrap'
+import { RadioGroup } from '../radio/RadioGroup'
+import './facet.scss'
+
+export interface FacetRadioProps {
+  radioArr: {
+    label: string
+    onChange?: React.MouseEventHandler<HTMLInputElement>
```

Comment:
For this component, it would be better to put one onChange listener around the whole group, rather than setting one per button. [This guide](https://www.pluralsight.com/guides/how-to-use-radio-buttons-in-reactjs) has a good example for that.

Reply:
Quick Question, there is a setActive for the whole group. I have this onChange for the My Dataset Radio that would conditionaly show the My Datasets Status dropdown.

Reply:
That should be good for now.